WooCommerce/WooCommerceTests/Mockups/MockPushNotificationsManager.swift

```diff
@@ -55,7 +55,7 @@ extension MockPushNotificationsManager {
     /// Send a ForegroundNotification that will be emitted by the `foregroundNotifications`
```

Comment:
nit: `ForegroundNotification ` -> `PushNotification`? 😄

Reply:
Thank you! Fixed in a499f5856fce27fa87ce56fd2edb87af1b58a54d